scripts/llama_summary.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+"""
+AI Summary generator for Tulsa Government Access Television meetings.
+
+This script processes meetings that need summarization in summaries.jsonl,
+and uses an AI model to generate summaries for them.
+"""
+import os
+import sys
+import json
+import asyncio
+from pathlib import Path
+from datetime import datetime, timezone
+from typing import Dict, List, Any, Optional
+import aiohttp
+from dotenv import load_dotenv
+
+
+# Add the parent directory to the path so we can import from src
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from src.models.meeting import MeetingSummary
+from collect_summaries import load_summaries
+from src.documents import read_pdf
+
+from collect_summaries import SUMMARY_FILE
+
+# Load environment variables from .env file
+load_dotenv()
+
+
+# GitHub AI Inference API endpoint (Llama-3.3-70B-Instruct)
+GITHUB_API_ENDPOINT = "https://models.inference.ai.azure.com/chat/completions"
+MODEL_NAME = "Llama-3.3-70B-Instruct"
+
+# Get GitHub token from environment variable
+GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
+
+
+async def get_summary_text(summary: MeetingSummary) -> str:
```

Comment:
question (non-blocking): this one at least really `await`s an async function. But I'm not sure why we're using `aiohttp`? Are we planning to do these all concurrently?